docs/docs/examples/model-inference/S3-Model-Inference/index.md

```diff
@@ -1,40 +1,47 @@
 ---
-sidebar_label: "S3-Model-Inference"
+sidebar_label: "Run Inference on a S3-stored Model"
 sidebar_position: 3
 ---
-# Running Inference on a Model stored on S3
+# Run Inference on a S3-stored Model
```

Comment:
"a S3" is hard to say and so this reads funny.  We should use "an" or perhaps find another way of saying it than S3-stored?

Reply:
I think the best solution is to go back to the original ` Run Inference on a model stored on S3` wording